web-client/public/gene/api.js

```diff
@@ -175,7 +175,19 @@ let getJasparInfo = function (geneSymbol) {
             xhr.setRequestHeader("content-type", "application/json");
         },
     }).then(function (data) {
-        return (data.count !== 1 ? {} :
+        let value = undefined;
+        const nameCheck = (list) => {
+            if (list !== []) {
```

Comment:
Note that `[] !== []` is `true`. If you are checking for an empty list, do something different.